autoload/rails.vim

```diff
@@ -4206,7 +4206,8 @@ function! s:app_db_url(...) dict abort
     endif
   endif
   if !empty(config)
-    let url .= '?' . join(map(items(config), 'v:val[0]."=".s:url_encode(v:val[1])'), '&')
+    let flat_config = filter(config, 'type(v:val) != type([]) && type(v:val) != type({})')
```

Comment:
`filter()` mutates, let's make this `call filter(...)`

Reply:
Gotcha 👍

Reply:
You did something different.